pkg/lightning/mydump/region.go

```diff
@@ -238,6 +239,9 @@ func MakeTableRegions(
 			)
 			dataFileSize := info.FileMeta.FileSize
 			if info.FileMeta.Type == SourceTypeParquet {
+				failpoint.Inject("mockMakeParquetFileRegion", func() {
+					time.Sleep(time.Second * 3)
+				})
```

Comment:
```suggestion
				failpoint.Inject("mockMakeParquetFileRegion", func() {})
```
failpoint expression can use `sleep` directly